skype/skype-ps/skype/Get-CsTenantNetworkRegion.md

```diff
@@ -0,0 +1,116 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+applicable: Skype for Business Online
+title: Get-CsTenantNetworkRegion
+schema: 2.0.0
+---
+
+# Get-CsTenantNetworkRegion
+
+## SYNOPSIS
+
+## SYNTAX
+
+```
+Get-CsTenantNetworkRegion [-LocalStore] [[-Identity] <Object>] [-Tenant <Object>] [-Filter <Object>] [-AsJob]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}

```

Comment:
You need to add descriptions, examples, etc.

Reply:
Engineers will add once it's checked in